app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -23,6 +36,75 @@
 
 public class BridgeUtils {
     
+    private static final Logger logger = LoggerFactory.getLogger(BridgeUtils.class);
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    /**
+     * Used for the Stormpath REST API. This should go away when they update their SDK.
+     * @param client
+     * @param url
+     * @param node
+     * @return
+     * @throws Exception
+     */
+    public static ObjectNode getJSON(CloseableHttpClient client, String url) throws Exception {
+        HttpGet get = new HttpGet(url);
+        get.setHeader("Accept", "application/json");
+        try(CloseableHttpResponse response = client.execute(get)) {
+            ObjectNode object = (ObjectNode)MAPPER.readTree(EntityUtils.toString(response.getEntity(), "UTF-8"));
+            if (logger.isDebugEnabled()) {
+                logger.debug("GET: " + url + "\n   request: <EMPTY>\n   response: "+ MAPPER.writeValueAsString(object));
+            }
+            return object;
+        }
+    }
+    
+    /**
+     * Used for the Stormpath REST API. This should go away when they update their SDK.
+     * @param client
+     * @param url
+     * @param node
+     * @return
+     * @throws Exception
+     */
+    public static ObjectNode postJSON(CloseableHttpClient client, String url, ObjectNode node) throws Exception {
+        HttpPost post = new HttpPost(url);
+        post.setHeader("Accept", "application/json");
+        post.setHeader("Content-Type", "application/json");
+        post.setEntity(new StringEntity(MAPPER.writeValueAsString(node), "UTF-8"));
+        
+        try (CloseableHttpResponse response = client.execute(post)) {
+            ObjectNode object = (ObjectNode)MAPPER.readTree(EntityUtils.toString(response.getEntity(), "UTF-8"));
+            if (logger.isDebugEnabled()) {
+                logger.debug("POST: " + url + "\n   request: "+MAPPER.writeValueAsString(node)+"\n   response: " + MAPPER.writeValueAsString(object));    
+            }
+            return object;
+        }
+    }
+    
+    public static String resolveTemplate(String template, Map<String,String> values) {
+        checkNotNull(template);
+        checkNotNull(values);
+        
+        for (String key : values.keySet()) {
+            String value = values.get(key);
+            if (value != null) {
+                String regex = "\\$\\{"+key+"\\}";
+                template = template.replaceAll(regex, values.get(key));
+            }
+        }
+        return template;
+    }
+    
+    public static String toStringQuietly(Resource resource) {
+        try {
+            return IOUtils.toString(resource.getInputStream());
+        } catch(IOException ioe) {
+            throw new RuntimeException(ioe);
```

Comment:
Wrapping things in RuntimeExceptions makes things harder to test (since we lose the ability to distinguish between this and, for example, a NullPointerException). If the goal is to not have to deal with checked exceptions, I recommend creating your own subclass of RuntimeException and throwing that instead.